SecurityPkg/Library/HashLibTpm2/HashLibTpm2PeilessSecLib.c

```diff
@@ -586,9 +586,7 @@ HasLibTpm2PeilessSecLibConstructor (
   }
 
   TcgPcrEvent             = (TCG_PCR_EVENT *)EventLog;
-  TcgEfiSpecIdEventStruct = (TCG_EfiSpecIDEventStruct *)
-                            (EventLog + OFFSET_OF (TCG_PCR_EVENT, Event));
-
+  TcgEfiSpecIdEventStruct = (TCG_EfiSpecIDEventStruct *)TcgPcrEvent->Event;

```

Comment:
Yeah better, thanks.

But please drop the URL from the commit log, as it will go stale very quickly, and no need for the stream-of-consciousness.